### DISCUSSION TITLE
coreutils should be a runtime dependency of idris2

### DISCUSSION THREAD

**Discussion by michaelmesser** [Homebrew/discussions#394] [Everyday usage]
https://github.com/idris-lang/Idris2/issues/874 Here is the [formula](https://github.com/Homebrew/homebrew-core/blob/b59c1118dbaa3f10540f525d7955cacd3fe24d3e/Formula/idris2.rb). I think the `depends_on "coreutils" => :build` should be `depends_on "coreutils"` but I am not certain. Can someone who understands homebrew formula take a look? I didn't experience this issue since I already had coreutils installed.

**Comment by SMillerDev** [comment]
Please make a pull request if you have suggestions for a change in homebrew.
